### PR TITLE
Copy lmai eval-cookbooks to this repo

### PR DESCRIPTION
**Pull request by rossdanlm**
Copy lmai eval-cookbooks to this repo

In future we should automate GH actions but for now this works. RAn this on getting started and it works
Screenshot 2024-07-19 at 11 32 34

### CODE REVIEW

File: tutorials/distributed_tracing.ipynb [hunk 1]
```diff
@@ -242,25 +242,27 @@
...
-        "    `rag-debug launch`\n",
-        "\n",
-        "Once you've completed these steps, click on the url provided to open the RAG Workbench UI. \n",
+        "Navigate to the url provided by the RAG Workbench (opens up your web browser). This will look like http://localhost:8080/\n",
         "\n",
         "1. Click the **Traces Tab**.\n",
         "2. Select Project **'Distributed Tracing'**.\n",
         "\n",
+        "TODO (tanya) - The traces aren't showing up under the Project - Distributed Tracing\n",
+        "\n",
```
Comment (rossdanlm) [inline tutorials/distributed_tracing.ipynb:265]: Should delete

File: tutorials/distributed_tracing.ipynb
Comment (rholinshead) [inline tutorials/distributed_tracing.ipynb:1]: We should undo the changes here since it's already on latest in this repo. The copy in lmai repo has not changed since it was added

File: tutorials/evaluate_llm_generated_pr_titles.ipynb [hunk 2]
```diff
@@ -514,21 +517,11 @@
         "\n",
         "## Step 6: View Evaluation Results\n",
         "\n",
-        "You can view the results in the RAG Workbench UI. Here's how to access the UI:\n",
-        "\n",
-        "1. Install the UI package by running the following command in your terminal:\n",
```
Comment (rholinshead) [inline tutorials/evaluate_llm_generated_pr_titles.ipynb:519]: Can we include the UI package installation here as well? It was added in this repo after the monorepo was made and not synced

File: tutorials/Quick_Start_RAG_Workbench.ipynb
Comment (rholinshead) [inline tutorials/Quick_Start_RAG_Workbench.ipynb:1]: Not sure we actually need this one here tbh, it seems incomplete / not intended for user share-out yet

File: tutorials/evaluation_metrics.ipynb
Comment (rholinshead) [inline tutorials/evaluation_metrics.ipynb:1]: Should undo this as well since it has been updated in this repo but not monorepo

File: tutorials/getting_started.ipynb
Comment (rholinshead) [inline tutorials/getting_started.ipynb:1]: Need to consolidate changes with https://github.com/lastmile-ai/eval-cookbook/commits/main/tutorials/getting_started.ipynb

File: tutorials/langchain/langchain_agent.ipynb [hunk 3]
```diff
@@ -287,22 +248,20 @@
         "<a name=\"step5\"></a>\n",
         "\n",
         "# Step 5: View Trace Data in RAG Debugger\n",
-        "You can view the results in the RAG Workbench UI. Here's how to access the UI:\n",
-        "\n",
-        "1. Install the UI package by running the following command in your terminal:\n",
```
Comment (rholinshead) [inline tutorials/langchain/langchain_agent.ipynb:292]: Will need to consolidate this UI package instruction into the new changes

File: tutorials/llamaindex/llamaindex_function_calls.ipynb [hunk 4]
```diff
@@ -247,21 +247,21 @@
         "\n",
         "## Step 5: View Results in RAG Workbench UI\n",
         "\n",
-        "You can view the results in the RAG Workbench UI. Here's how to access the UI:\n",
+        "We can view the results in the RAG Workbench UI.\n",
         "\n",
-        "1. Install the UI package by running the following command in your terminal:\n",
```
Comment (rholinshead) [inline tutorials/llamaindex/llamaindex_function_calls.ipynb:252]: Same here r.e. install instructions for UI

File: tutorials/llamaindex/llamaindex_quickstart.ipynb [hunk 5]
```diff
@@ -255,21 +239,21 @@
         "<a name=\"step6\"></a>\n",
         "\n",
         "# Step 6: View Results in RAG Workbench UI\n",
-        "You can view the results in the RAG Workbench UI. Here's how to access the UI:\n",
-        "\n",
-        "1. Install the UI package by running the following command in your terminal:\n",
```
Comment (rholinshead) [inline tutorials/llamaindex/llamaindex_quickstart.ipynb:260]: and here

File: tutorials/parameter_experimentation.ipynb [hunk 6]
```diff
@@ -436,21 +402,17 @@
         "\n",
         "## Step 5: View Evaluation Results\n",
         "\n",
-        "You can view the results in the RAG Workbench UI. Here's how to access the UI:\n",
-        "\n",
-        "1. Install the UI package by running the following command in your terminal:\n",
-        "\n",
-        "    `pip install \"lastmile-eval[ui]”`\n",
```
Comment (rholinshead) [inline tutorials/parameter_experimentation.ipynb:443]: same here

File: tutorials/user_feedback.ipynb [hunk 7]
```diff
@@ -334,21 +334,21 @@
       "source": [
         "<a name=\"view\" id=\"view\"></a>\n",
         "## View user feedback results\n",
-        "You can view the results in the RAG Workbench UI. Here's how to access the UI:\n",
+        "We can view the results in the RAG Workbench UI.\n",
         "\n",
-        "1. Install the UI package by running the following command in your terminal:\n",
```
Comment (rholinshead) [inline tutorials/user_feedback.ipynb:339]: and here

**Review by rholinshead — CHANGES_REQUESTED** [review]
Requesting some changes, mostly minor updates to current repo that should be included (e.g. UI package install instructions). Some others that should not be copied over as well

File: tutorials/Quick_Start_RAG_Workbench.ipynb
Comment (rossdanlm) [reply tutorials/Quick_Start_RAG_Workbench.ipynb:1]: Tanya says to keep it and make it public facing

File: tutorials/langchain/langchain_agent.ipynb [hunk 8]
```diff
@@ -141,25 +130,16 @@
...
-          ]
-        }
-      ],
+      "outputs": [],
       "source": [
         "from tracing_auto_instrumentation.langchain import LangChainInstrumentor\n",
         "\n",
         "# Create an instance of LangChainInstrumentor and instrument the code\n",
-        "instrumentor = LangChainInstrumentor(project_name=\"Plan-and-Execute Example\")\n",
+        "instrumentor = LangChainInstrumentor(project_name=\"LangChain-Example\")\n",
```
Comment (rossdanlm) [inline tutorials/langchain/langchain_agent.ipynb:142]: cc @tanya-rai do we still want to call this `"Plan-and-Execute"?

File: tutorials/Quick_Start_RAG_Workbench.ipynb [hunk 9]
```diff
@@ -0,0 +1,115 @@
+{
```
Comment (rossdanlm) [inline tutorials/Quick_Start_RAG_Workbench.ipynb:1]: cc @tanya-rai said to keep this in public repo

File: tutorials/Quick_Start_RAG_Workbench.ipynb
Comment (tanya-rai) [reply tutorials/Quick_Start_RAG_Workbench.ipynb:1]: Here;s my reasoning - Quick Start in the actual documentation (rag.lastmileai.dev) needs to be updated. I can't keep track of the API changes so I want to make sure that when we make those changes we update the notebook here so I can update the documentation page as well.

**Review by rholinshead — APPROVED** [review]
Thanks!

File: tutorials/Quick_Start_RAG_Workbench.ipynb
Comment (rossdanlm) [reply tutorials/Quick_Start_RAG_Workbench.ipynb:1]: WE should really just have automated testing instead but for now we can keep. More short term, after I get the GH workflow actions to clone repo, we can move the lmai copy to deprecated
